### PR TITLE
Add initialization of the file system for the orchestrator

### PR DESCRIPTION
**Pull request by tdrozdovsky**
Signed-off-by: Taras Drozdovskyi <t.drozdovsky@samsung.com>

# Description

If the file system required to work of edge-orchestration is not yet created, then it creates file system. 

## Type of change

- [x] Code cleanup/refactoring
- [x] Documentation update

# How Has This Been Tested?

Run the `./build.sh` command on a clean system 
After startup, you will see the file system:

```
├── apps
├── data
│   ├── cert
│   └── db
├── datastorage
├── device
├── log
├── mnedc
└── user
```

**Test Configuration**:
* Firmware version: Ubuntu 18.04, etc.)
* Hardware: x86-64, ARM
* Toolchain: Docker and Go recommended versions
* Edge Orchestration Release: Coconut

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: cmd/edge-orchestration/main.go [hunk 1]
```diff
@@ -69,8 +70,21 @@ const (
 	cipherKeyFilePath = edgeDir + "/user/orchestration_userID.txt"
 	deviceIDFilePath  = edgeDir + "/device/orchestration_deviceID.txt"
 	mnedcServerConfig = edgeDir + "/mnedc/client-config.yaml"
+
+	orchUserID = "Home Edge Network\n"
```
Comment (t25kim) [inline cmd/edge-orchestration/main.go:74]: Is it okay if we open this cipher key in public?

**Review by t25kim — COMMENTED** [review]
Thanks for your contribution!
It seems that the same code is being repeated for each language such as c, java, go. How about developing the function under internal/common/{something} and calling the function for each language to maintain easy when changing files or folders in the future.

File: cmd/edge-orchestration/main.go (hunk 1 above)
Comment (tdrozdovsky) [reply cmd/edge-orchestration/main.go]: Done!

**Review by t25kim — APPROVED** [review]
Awesome! Thanks!

File: cmd/edge-orchestration/main.go (hunk 1 above)
Comment (tdrozdovsky) [reply cmd/edge-orchestration/main.go]: Done!

**Review by MoonkiHong — APPROVED** [review]
+1, Thank you for your contribution!

**Comment by MoonkiHong** [comment]
@suresh-lc PTAL. 😄

**Pull request merged**